toolkit/tools/osmodifier/main.go

```diff
@@ -47,6 +47,15 @@ func main() {
 }
 
 func modifyImage() error {
+	if *configFile == "" {
```

Comment:
It would probably be better to use [subcommands](https://github.com/alecthomas/kingpin?tab=readme-ov-file#sub-commands) than keying off if the `--config-file` arg is provided or not.

Reply:
I've tried subcommands, it will somehow fail to read flags (--config-file), so I'll keep them both as flags